packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -339,7 +340,7 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
     }
 
     const { onRenderDetailsHeader = this._onRenderDetailsHeader } = this.props;
-
+    const recomputedGroupProps = groupProps ? this._getGroupProps(groupProps) : undefined;
```

Comment:
>const recomputedGroupProps = groupProps ? this._getGroupProps(groupProps) : undefined; [](start = 4, length = 86)

Is this supposed to be used only by GroupedList? If yes, we should move it inline to avoid confusion elsewhere b/w groupProps and recomputedGroupProps.